### ISSUE TITLE
invalid operator-enclosed-by-spaces error

### ISSUE THREAD

**Issue by ryanmerolle** [aristanetworks/j2lint#169]
## Current Operation

```jinja
{% if blah %}commands all default stop-only group tacacs+{% endif %}
```

```bash
j2lint test.j2 --ignore single-statement-per-line
───────────────────────────────────────────────────────────── JINJA2 LINT ERRORS ──────────────────────────────────────────────────────────────
test.j2
└── test.j2:1 The operator + needs to be enclosed by a single space on each side (operator-enclosed-by-spaces)

Jinja2 linting finished with 1 error(s) and 0 warning(s)
```

## How it Should Work

it should only evaluate this rule within Jinja's code blocks.

**Comment by gmuloc** [comment]
Hi @ryanmerolle indeed this is not a usecase we hit in AVD because we enforce a single statement per line which you are disabling explicitly in your command. I would potentially expect other rules to fail given how they were built initially.

This would require to adjust the rule parsing, maybe at the cost of performance so would need to take a look into that.
